tapdb/sqlc/queries/transfers.sql

```diff
@@ -194,3 +194,33 @@ FROM passive_assets as passive
     JOIN managed_utxos utxos
         ON passive.new_anchor_utxo = utxos.utxo_id
 WHERE passive.transfer_id = @transfer_id;
+
+-- name: InsertBurn :one
+INSERT INTO asset_burn_transfers (
+    transfer_id, note, asset_id, group_key, amount
+)
+VALUES (
+    @transfer_id, @note, @asset_id, @group_key, @amount
+)
+RETURNING burn_id;
+
+-- name: QueryBurns :many
+SELECT
+    abt.note,
+    abt.asset_id,
+    abt.group_key,
+    abt.amount,
+    ct.txid AS anchor_txid -- Retrieving the txid from chain_txns
+FROM asset_burn_transfers abt
+JOIN asset_transfers at ON abt.transfer_id = at.id
```

Comment:
Looks like we don't need to join on `asset_transfers`? Since we don't seem to be using `at` anywhere.

Reply:
Perhaps the idea was to return the full transfer information alongside the burn? 

From the PoV of a future query CLI, I think that would be desirable.

Reply:
I need to join in order to grab the asset_transfers row, and then link to the chain_txns row to get the txid